static/js/containers/ChannelPage.js

```diff
@@ -98,12 +82,10 @@ const mapStateToProps = (state, ownProps) => {
     channelName,
     channel,
     posts:              safeBulkGet(postIds || [], state.posts.data),
-    subscribedChannels: state.subscribedChannels.loaded
-      ? safeBulkGet(state.subscribedChannels.data, state.channels.data)
-      : null,
-    loaded:  R.none(R.isNil, [channel, postIds]),
-    errored: anyError([state.channels, state.posts, state.subscribedChannels])
+    subscribedChannels: getSubscribedChannels(state),
+    loaded:             R.none(R.isNil, [channel, postIds]),
+    errored:            anyError([state.channels, state.posts, state.subscribedChannels])
   }
 }
 
-export default R.compose(connect(mapStateToProps), withLoading)(ChannelPage)
+export default R.compose(connect(mapStateToProps), withNavSidebar, withLoading)(ChannelPage)
```

Comment:
I would like to as well, but since we're using the source-code formatter we are stuck with this unless we decided to change the line width. lines like this are why I made this proposal: https://github.com/mitodl/open-discussions/issues/120

Reply:
Can you link me to the documentation to whatever this source-code formatter is? I'm really confused about why this wouldn't be allowed

Reply:
It's called [prettier](https://prettier.io/), we started using it open-discussions a few weeks ago. I like it a lot so far, although the only thing I'm not pleased with is the amount of code that ends up on single lines when they are set to use a 120-character width.

If you want to play around with it you can use the formatter in this project by doing `npm run fmt`.

Reply:
We're not using the main project directly, instead we're using a thing to get it to work nicely with eslint called [eslint-prettier-cli](https://github.com/prettier/prettier-eslint-cli)